packages/atomic/src/locales.json

```diff
@@ -801,6 +801,10 @@
     "zh-cn": "{{value}} 更多...",
     "zh-tw": "{{value}} 更多..."
   },
+  "showNMoreFilters": {
```

Comment:
What string did we use for JSUI, and can we re-use/re-import that string with all translation ?